src/wallet/rpc/addresses.cpp

```diff
@@ -640,14 +640,15 @@ RPCHelpMan getaddressesbylabel()
     return RPCHelpMan{"getaddressesbylabel",
                 "\nReturns the list of addresses assigned the specified label.\n",
```

Comment:
If you want to do this, you should mention that if no label is supplied then all address book entry addresses are returned

Reply:
Done in 37f809aad549368911b8358c1c6e6cb0e8c5bc7e. Thanks.